webplugin/js/app/zendesk-chat-service.js

```diff
@@ -324,7 +324,6 @@ var onTabClickedHandlerForZendeskConversations = function (event) {
         })
         if (!newConversationCreated && assigneeInfo.role != KommunicateConstants.GROUP_ROLE.MODERATOR_OR_BOT) {
             newConversationCreated = true;
-            Kommunicate.startConversation();
```

Comment:
i think we need this code, its because in zendesk we cant continue last conversation.
the bug you are looking to fix is not this one.
cross check once if zendesk is working fine with this or not

Reply:
The bug I was trying to fix was coming because of this line only. The bug is fixed after the removal of this line.

And when are we not suppose continue with the last conversation? After chat-widget reload?

Reply:
we cant continue last conversation in zendesk. its how their sdk was developed. hence this code.

after this change, are you able to continue last conversation in both direction ? zendesk - kommunicate

Reply:
No I am not being able to continue with last conversation in both direction. 

I am trying after chat resolution or/and chat-widget reload. In both cases last convo is not coming on widget.